### PR TITLE
python-modules: fix httpbin by adding all the deps

### LINKED ISSUE #33604: brotlipy not building
## Issue description

```
Copying /nix/store/w7s57pzrb527vav1hcqmx4cjblhw6a23-zfs-user-2017-12-28/lib/udev/rules.d/60-zvol.rules to /nix/store/h6ncci4gxsfvyxd8p06lmcq932xpsig4-udev-rules/60-zvol.rules
Copying /nix/store/w7s57pzrb527vav1hcqmx4cjblhw6a23-zfs-user-2017-12-28/lib/udev/rules.d/69-vdev.rules to /nix/store/h6ncci4gxsfvyxd8p06lmcq932xpsig4-udev-rules/69-vdev.rules
Copying /nix/store/w7s57pzrb527vav1hcqmx4cjblhw6a23-zfs-user-2017-12-28/lib/udev/rules.d/90-zfs.rules to /nix/store/h6ncci4gxsfvyxd8p06lmcq932xpsig4-udev-rules/90-zfs.rules
Adding rules for package /nix/store/m87ghm01a15xdbv5q8c5zilpp1hnrack-bcache-tools-1.0.7
Copying /nix/store/m87ghm01a15xdbv5q8c5zilpp1hnrack-bcache-tools-1.0.7/lib/udev/rules.d/69-bcache.rules to /nix/store/h6ncci4gxsfvyxd8p06lmcq932xpsig4-udev-rules/69-bcache.rules
Adding rules for package /nix/store/h6vbviyvj7zkc451lhi0jgqnkq2n4ph8-udisks-2.1.6
Copying /nix/store/h6vbviyvj7zkc451lhi0jgqnkq2n4ph8-udisks-2.1.6/lib/udev/rules.d/80-udisks2.rules to /nix/store/h6ncci4gxsfvyxd8p06lmcq932xpsig4-udev-rules/80-udisks2.rules
Adding rules for package /nix/store/ffgywglj8f2gp88xxk5hgs87b0zly6zj-extra-udev-rules
Copying /nix/store/ffgywglj8f2gp88xxk5hgs87b0zly6zj-extra-udev-rules/etc/udev/rules.d/99-local.rules to /nix/store/h6ncci4gxsfvyxd8p06lmcq932xpsig4-udev-rules/99-local.rules
building path(s) ‘/nix/store/4x1idl4akbjci5nszzqc3dcfgqicmvbw-ui_pyroscope.patch’
Adding rules for package /nix/store/ph965c18yp2802lx95qsxxh923sc8vl2-extra-hwdb-file
Adding rules for package /nix/store/kb8f2nldxdypyv8vywv8isd9sd49yyzb-alsa-utils-1.1.5
Copying /nix/store/kb8f2nldxdypyv8vywv8isd9sd49yyzb-alsa-utils-1.1.5/lib/udev/rules.d/90-alsa-restore.rules to /nix/store/h6ncci4gxsfvyxd8p06lmcq932xpsig4-udev-rules/90-alsa-restore.rules
building path(s) ‘/nix/store/v09hxqwhp2m5zva8ysbdismbm1sm5m4x-unit-alsa-store.service’
Checking that all programs called by relative paths in udev rules exist in /nix/store/j8nxyspnpkw0w3mzp24ba9jzzqpzdw48-systemd-234/lib/udev... OK
Checking that all programs called by absolute paths in udev rules exist... OK
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
configure.ac:3: installing './compile'
configure.ac:20: installing './missing'
building path(s) ‘/nix/store/3c752zi84smf586c1iyd8j2gimmc89wb-stage-1-init.sh’
src/Makefile.am: installing './depcomp'
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0building path(s) ‘/nix/store/basma31ng1i47109fmgi5wfqp4x1j1x4-unit-audit.service’
Processing ./httpbin-0.6.2-py2.py3-none-any.whl
Collecting brotlipy (from httpbin==0.6.2)
  Could not find a version that satisfies the requirement brotlipy (from httpbin==0.6.2) (from versions: )
No matching distribution found for brotlipy (from httpbin==0.6.2)
builder for ‘/nix/store/948kf5ismkxwn1cj1r5gfn83f26p8xc7-python2.7-httpbin-0.6.2.drv’ failed with exit code 1
cannot build derivation ‘/nix/store/qfmb1a0gc5l1x1504679gjhpsfwbwv51-python2.7-pytest-httpbin-0.2.3.drv’: 1 dependencies couldn't be built
building path(s) ‘/nix/store/2ppibhjzfmcpa0k8jpjvm4ph6455sdd8-unit-console-getty.service-disabled’
cannot build derivation ‘/nix/store/81cjdmca9abdm56wzdgy02904lxz1a4i-python2.7-vcrpy-1.11.1.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/acj7p6jbfswsqsq4knaw8h1z0a4vkdwj-FlexGet-2.10.82.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/cwvmxp0dx7w4yss3x3rr3gchbrm91fix-system-path.drv’: 1 dependencies couldn't be built
building path(s) ‘/nix/store/698ik52qcsv9v89q8viq1pp8krkkydp0-unit-container-getty-.service’
building path(s) ‘/nix/store/n50vq0lf8qxkfiyzn7r6nszsd03sr7ny-unit-cpufreq.service’
building path(s) ‘/nix/store/rcwdm3781ywy7a66xxrl3d7pcprmv1q7-unit-dbus.socket’
building path(s) ‘/nix/store/71naa0a8zb8sckky2psfn3pwv06dhi42-unit-dhcpcd.service’
building path(s) ‘/nix/store/9xd7vacg4d66jy88wahd8r9p3dbqy4ki-unit-firewall.service’
cannot build derivation ‘/nix/store/16a9a7s99km7z4qw1nif3nycp3s9qqwd-unit-flexget.service.drv’: 1 dependencies couldn't be built
building path(s) ‘/nix/store/9k1hhpz9mc7hprpx6bnlgxda29l86s24-unit-fs.target’
building path(s) ‘/nix/store/jkvlljdcwd14l4cdb5yqxigxw28y336l-unit-getty-.service’
building path(s) ‘/nix/store/ad2nbf30vycfvyvgf32vgrg0gp19mlmj-unit-graphical-session.target’
cannot build derivation ‘/nix/store/kdfr5fsamillv6m8j5jps724v4z0pxik-nixos-system-servi-18.03pre124914.b9806751074.drv’: 1 dependencies couldn't be built
error: build of ‘/nix/store/kdfr5fsamillv6m8j5jps724v4z0pxik-nixos-system-servi-18.03pre124914.b9806751074.drv’ failed
```

### Steps to reproduce

No idea which package requires brotlipy

## Technical details

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.

 - system: `"x86_64-linux"`
 - host os: `Linux 4.9.72, NixOS, 18.03pre123890.9d4de1eac81 (Impala)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 1.11.16`
 - channels(hyper): `""`
 - channels(root): `"nixos-18.03pre124914.b9806751074"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`

### PR DESCRIPTION
**Pull request by dtzWill**
###### Motivation for this change

Fixes #33604 .

Fixes httpbin build failure.
Introduces number of new packages that are required by httpbin as of 0.6.2.

httpbin should not have [been updated](https://github.com/NixOS/nixpkgs/commit/0a447a6a3da3087b9ca6ebf3ce1eb277d41cdebd) without testing that it builds....

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/crayons/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchPypi, buildPythonPackage, colorama }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
```
Comment (FRidh) [inline pkgs/development/python-modules/crayons/default.nix]: drop name here and in other expressions

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/crayons/default.nix (hunk 1 above)
Comment (dtzWill) [reply pkgs/development/python-modules/crayons/default.nix]: Oh, that's automatic? Excellent!

**Comment by dtzWill** [comment]
@FRidh: done, pushed!

**Comment by sjau** [comment]
any news on this?

**Comment by dtzWill** [comment]
Rebased onto master and re-tested, ping! (cc @FRidh)

Since the package (and anything requiring it) currently fails to build, this seems moderately important.  And with a low bar to clear in terms of functionality :P.

File: pkgs/development/python-modules/crayons/default.nix (hunk 1 above)
Comment (orivej) [reply pkgs/development/python-modules/crayons/default.nix]: Ping!

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -8035,12 +8037,16 @@ in {
...
 
+  flask_common = callPackage ../development/python-modules/flask-common { };
+
   flask-compress = callPackage ../development/python-modules/flask-compress { };
 
   flask-cors = callPackage ../development/python-modules/flask-cors { };
 
   flask_elastic = callPackage ../development/python-modules/flask-elastic { };
 
+  flask_limiter = callPackage ../development/python-modules/flask-limiter { };
```
Comment (orivej) [inline pkgs/top-level/python-packages.nix]: Since the canonical PyPI names of the new packages are Flask-Common and Flask-Limiter, and the manual says not to convert hyphens to underscores (https://nixos.org/nixpkgs/manual/#sec-package-naming), could you use hyphenated names?

File: pkgs/development/python-modules/pytzdata/default.nix [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1wi3jh39zsa9iiyyhynhj7w5b2p9wdyd0ppavpsrmf3wxvr7cwz8";
+  };
+
+  # No tests
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "Official timezone database for Python";
```
Comment (orivej) [inline pkgs/development/python-modules/pytzdata/default.nix]: This is the description on its website, but I do not see anything official about it. Could you change it to "Timezone database for Python"?

It is quite unfortunate that it ships an outdated version of tzdata rather than using system tzdata, but this is out of scope of this PR.

File: pkgs/top-level/python-packages.nix (hunk 2 above)
Comment (FRidh) [reply pkgs/top-level/python-packages.nix]: Actually, in `python-packages.nix` we do not follow those guidelines, but instead stick to the canonical PyPI names, or the normalized names (https://www.python.org/dev/peps/pep-0503/#normalized-names)

Comment (FRidh) [reply pkgs/top-level/python-packages.nix]: At least, that's what I am pushing for.

**Comment by FRidh** [comment]
I've included the changes in https://github.com/NixOS/nixpkgs/pull/34077.
